templates/components/form/item_remote_management.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends "generic_show_form.html.twig" %}
+{% import 'components/form/fields_macros.html.twig' as fields %}
+{% block more_fields %}
+    <input type='hidden' name='items_id' value='{{options['items_id']}}'>
+    <input type='hidden' name='itemtype' value='{{options['itemtype']}}'>
```

Comment:
Please use double quotes for html attributes. It does not change much, but it is easier to read when all the code follows the same standards.